migrations/artifact_tenant_token_migration.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python3
```

Comment:
One time use script, not sure if this should be committed into this repo.

Reply:
I can drop it, and keep it locally. It was worth getting review though :)